tests/test-chat-template.cpp

```diff
@@ -133,6 +133,16 @@ int main(void) {
     std::vector<char> formatted_chat(1024);
     int32_t res;
 
+    // list all supported templates
+    std::vector<const char *> supported_tmpl(1024);
```

Comment:
```suggestion
	std::vector<const char *> supported_tmpl;
    res = llama_chat_builtin_templates(nullptr, 0);
    supported_tmpl.resize(res);
```

Reply:
fixed in https://github.com/ggerganov/llama.cpp/pull/10623/commits/86ff8e39294733752d08ff64a04aac6b80b66005